geomesa-spark/geomesa-spark-geotools/src/test/scala/org/locationtech/geomesa/spark/geotools/GeoToolsSpatialRDDProviderTest.scala

```diff
@@ -0,0 +1,66 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark.geotools
+
+import com.vividsolutions.jts.geom.Coordinate
+import org.apache.hadoop.conf.Configuration
+import org.apache.spark.{SparkConf, SparkContext}
+import org.geotools.data.simple.SimpleFeatureStore
+import org.geotools.data.{DataStore, DataStoreFinder, DataUtilities, Query}
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.joda.time.format.ISODateTimeFormat
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.spark.{GeoMesaSpark, GeoMesaSparkKryoRegistrator}
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class GeoToolsSpatialRDDProviderTest extends Specification {
+
+  "The GeoToolsSpatialRDDProvider" should {
+    "read from the in-memory database" in {
+      val dsParams = Map("cqengine" -> "true", "geotools" -> "true")
+      val ds = DataStoreFinder.getDataStore(dsParams)
+      ingestChicago(ds)
+
+      val conf = new SparkConf().setMaster("local[2]").setAppName("testSpark")
+
+      conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+      conf.set("spark.kryo.registrator", classOf[GeoMesaSparkKryoRegistrator].getName)
```

Comment:
NB: To use this module, I found that I have to set up the GMSpark Kryo Registrator like this.

@tkunicki got any better approaches?